src/lib/patternfly.sed

```diff
@@ -0,0 +1,2 @@
+s/src: url(.*eot[^'"]*['"]);$//
```

Comment:
Let's add a comment to that file why we need it, and where the original source is (cockpit's pkg/lib/)